qa/L0_orca/test.sh

```
@@ -138,6 +145,42 @@ function kill_server {
     done
 }
 
+function clone_tensorrt_llm_backend_repo {
+    rm -rf $TENSORRTLLM_BACKEND_DIR && mkdir $TENSORRTLLM_BACKEND_DIR
+    apt-get update && apt-get install git-lfs -y --no-install-recommends
+    git clone --single-branch --depth=1 -b ${TENSORRTLLM_BACKEND_REPO_TAG} ${TRITON_REPO_ORG}/tensorrtllm_backend.git $TENSORRTLLM_BACKEND_DIR
+    cd $TENSORRTLLM_BACKEND_DIR && git lfs install && git submodule update --init --recursive
+}
+
+function build_gpt2_base_model {
+    # Download weights from HuggingFace Transformers
+    cd ${GPT_DIR} && rm -rf gpt2 && git clone https://huggingface.co/gpt2-medium gpt2 && cd gpt2
+    rm pytorch_model.bin model.safetensors
+    if ! wget -q https://huggingface.co/gpt2-medium/resolve/main/pytorch_model.bin; then
+        echo "Downloading pytorch_model.bin failed."
+        exit 1
+    fi
+    cd ${GPT_DIR}
+
+    # Convert weights from HF Tranformers to FT format
+    python3 convert_checkpoint.py --model_dir gpt2 --dtype float16 --tp_size ${NUM_GPUS} --output_dir "./c-model/gpt2/${NUM_GPUS}-gpu/"
+    cd ${BASE_DIR}
+}
+
+function build_gpt2_tensorrt_engine {
+    # Build TensorRT engines
+    cd ${GPT_DIR}
+    trtllm-build --checkpoint_dir "./c-model/gpt2/${NUM_GPUS}-gpu/" \
+        --gpt_attention_plugin float16 \
+        --remove_input_padding enable \
+        --paged_kv_cache enable \
+        --gemm_plugin float16 \
+        --workers "${NUM_GPUS}" \
+        --output_dir "${ENGINES_DIR}"
+
+    cd ${BASE_DIR}
+}
+
 clone_tensorrt_llm_backend_repo
 build_gpt2_base_model
 build_gpt2_tensorrt_engine
```

How did this call to `build_gpt2_tensorrt_engine` previously work if the function didn't exist until this PR?

---

I tried finding an answer to this, but was unable to
Checked with @mc-nv if anything changed in the way CI runs or mounts. No luck.
Double checked if the test did actually run as expected before it broke (It did)
Test started breaking when we merged 24.04 release branch into main.

---

> They were originally ONLY defined in L0_perf_tensorrt_llm/test.sh and prev tests used to pick the defn from there

Do we do a `source L0_perf_tensorrt_llm/test.sh` somewhere when running this test?

---

Maybe because there is no `+e` so bash just ignore the error.